theme/static/js/hs-vue/left-header-app.js

```diff
@@ -555,15 +562,22 @@ let leftHeaderApp = new Vue({
     },
     filters: {
         nameWithoutCommas: function (name) {
-            if (!name) return '';
-            name = name.toString();
+            if (!name) return null;
+            name = name.toString().trim();
 
             if (name.indexOf(',') >= 0) {
                 let  fullName = name.split(',');
                 if (fullName.length == 2) {
                     let firstNames = fullName[1].trim();
                     let lastNames = fullName[0].trim();
-                    return firstNames + " " + lastNames;
+                    if (firstNames && lastNames)
```

Comment:
+ implies type coercion. If a value is null then I believe String() of that null would be 'null'. I believe this is not a possible scenario in the code, as written, though.

Reply:
if not, you open yourself to "null null"

Reply:
"null null" isn't possible here, the values come from name.split above.

Reply:
Well, the if (firstNames && lastNames) is what ensures that. I had been thinking along the lines of general best practices to follow. Thinking about it more, from a pure JS standpoint + should be fine and in some cases potentially better. In Discover work, I went with string interpolation, which is just like Python only syntactically slightly different.